ui-common/src/components/EntryDetailed/EntryViewer/AutoRepresentation.tsx

```diff
@@ -14,15 +14,14 @@ export enum TabsEnum {
     Response = 1
 }
 
-export const AutoRepresentation: React.FC<any> = ({ representation, color, defaultTab = TabsEnum.Request, isDisplayReplay = false }) => {
+export const AutoRepresentation: React.FC<any> = ({ representation, color, openedTab = TabsEnum.Request, isDisplayReplay = false }) => {
     const entryData = useRecoilValue(entryDataAtom)
     const setIsOpenRequestModal = useSetRecoilState(replayRequestModalOpenAtom)
     const isReplayDisplayed = useCallback(() => {
         return enabledProtocolsForReplay.find(x => x === entryData.protocol.name) && isDisplayReplay
     }, [entryData.protocol.name, isDisplayReplay])
 
-    const tabSelectedRef = useRef(defaultTab)
-
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
what is that for?

Reply:
suppress the dependency es-lint error

Reply:
why are you not adding “enabledProtocolsForReplay” to the deps array? 
I guess it would solve the lint error

Reply:
The 'TABS' array makes the dependencies of useCallback Hook (at line 35)